src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -14,14 +14,14 @@ import { useSwapper } from '../useSwapper/useSwapper'
 const ETHEREUM_ASSET_ID = 'eip155:1/slip44:60'
 
 export const useTradeRoutes = (
-  defaultBuyAssetId?: AssetId,
+  routeBuyAssetId?: AssetId,
```

Comment:
This name trips me out. `buyAssetIdRoute`?

Reply:
the buy asset id is from the route ultimately. defi suggested this name

Reply:
Ah, that's the origin - got it.